stix_shifter_modules/gcp_chronicle/requirements.txt

```diff
@@ -1 +1,2 @@
+# Consider using https://pypi.org/project/aiogoogle/ for async
 google-api-python-client==2.52.0
```

Comment:
were you able to try `aiogoogle` package?

Reply:
Sure, I was thinking about it. I would prefer a separate task for this

Reply:
https://github.com/opencybersecurityalliance/stix-shifter/issues/1220